src/omv/modules/py_ml.c

```diff
@@ -121,33 +121,45 @@ static mp_obj_t py_ml_process_output(py_ml_model_obj_t *model) {
     for (size_t i = 0; i < model->outputs_size; i++) {
         void *model_output = ml_backend_get_output(model, i);
         size_t size = py_ml_tuple_sum(MP_OBJ_TO_PTR(model->output_shape->items[i]));
-        mp_obj_tuple_t *output = MP_OBJ_TO_PTR(mp_obj_new_tuple(size, NULL));
+        mp_obj_tuple_t *output_shape = MP_OBJ_TO_PTR(model->output_shape->items[i]);
         float output_scale = mp_obj_get_float(model->output_scale->items[i]);
         int output_zero_point = mp_obj_get_int(model->output_zero_point->items[i]);
         int output_dtype = mp_obj_get_int(model->output_dtype->items[i]);
 
+        size_t shape[ULAB_MAX_DIMS] = {};
+
+        if (ULAB_MAX_DIMS < output_shape->len) {
+            mp_raise_msg(&mp_type_ValueError, MP_ERROR_TEXT("Output shape has too many dimensions"));
```

Comment:
I tried to put this in the tflm_backend.c file, but, there's no access to ULAB_MAX_DIMS in that file because DULAB_CONFIG_FILE is missing from the compile args.

Reply:
#include "ulab_config.h" ?

Reply:
Actually it should be here.